<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QVTKOpenGLNativeWidget" name="view_3d">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QVTKOpenGLNativeWidget" name="view_coronal">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QVTKOpenGLNativeWidget" name="view_sagittal">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QVTKOpenGLNativeWidget" name="view_axial">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>221</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Open DICOM</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKOpenGLNativeWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>QVTKOpenGLNativeWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
